app/services/deploy_service.rb

```diff
@@ -61,6 +76,20 @@ def github_access_token
       (raise 'A profile and basic_password are required for Github authentication')
   end
 
+  def deliver_slack_webhook(pull_request, webhook_url, merge_at)
+    uri = URI.parse webhook_url
+    request = Net::HTTP::Post.new(uri.request_uri)
+    request['Content-Type'] = 'application/json'
+    request.body = {
+      text: "The following changes will be released in #{distance_of_time_in_words_to_now(merge_at)}: #{pull_request.html_url}"
```

Comment:
Used [distance_of_time_in_words_to_now](https://api.rubyonrails.org/classes/ActionView/Helpers/DateHelper.html#method-i-distance_of_time_in_words_to_now) to specify the amount of time in friendly language, even if custom.